esileapclient/v1/offer.py

```diff
@@ -99,3 +101,17 @@ def delete(self, offer_uuid):
         """
 
         self._delete(resource_id=offer_uuid)
+
```

Comment:
so much simpler than the original code for creating and deleting. The reason the rest of this code is like this is because it is modelled after Ironic's client and doing it this way allows the client to be used as an api to manipulate 'esi' objects with python code. I guess we don't care about that or consistency with the rest of the code here?

Reply:
Oh - in this case it's because 'claim' shouldn't go in the base class because it's an offer-only operation. You're right that this makes our paradigm look a little funny; I was going to look into possibly cleaning things up after this patch got merged.

Reply:
its probably fine. The way things are structured is probably pointlessly complex as it doesn't do the things Ironic does in the end anyways. Basically a bunch of half measures that kind of futureproofs ourselves.